sqlglot/dialects/bigquery.py

```diff
@@ -34,12 +34,16 @@ def func(self, expression):
 
 
 class BigQuery(Dialect):
-    identifiers = ["`"]
-    escape = "\\"
     unnest_column_only = True
 
     class Tokenizer(Tokenizer):
-        QUOTES = ["'", '"', '"""']
+        QUOTES = [
+            quote if len(prefix) == 0 else (prefix + quote, quote)
```

Comment:
if prefix else

in python "" is falsey

Reply:
Fixed